cordlib/src/main/kotlin/com.luxoft.blockchainlab.corda.hyperledger.indy/flow/b2b/VerifyCredentialFlowB2B.kt

```diff
@@ -89,7 +89,7 @@ object VerifyCredentialFlowB2B {
                             proof,
                             usedData
                         )
-                    ) throw InvalidProofException("Proof verification failed")
+                    ) throw FlowException("Proof verification failed")
```

Comment:
FlowException is Corda' exception to close all connections with other participants. In my understanding it can be thrown on another side.
BTW let's merge it, but it need to be checked in a next loop

Reply:
It was here before this PR.